### ISSUE TITLE
Out-of-the-box interactivity

### ISSUE THREAD

**Issue by diegozea** [queryverse/VegaLite.jl#405]
Hi! Is it possible to get interactivity (particularly panning and zooming) enabled by default? Best,

**Comment by diegozea** [comment]
P.S.: Right now I can use `selection={grid={type=:interval, bind=:scales}}` to gain that functionality [*[ref]*](https://vda-lab.github.io/visualisation-tutorial/vegalite-zooming-and-panning.html).

**Comment by mcmcgrath13** [comment]
Panning and zooming doesn't make sense for all chart types, so my vote would be to keep it opt in.  Maybe add a utility to make it easier to opt into?

**Comment by diegozea** [comment]
Yes, that sounds sensible :)

**Comment by diegozea** [comment]
I thought a section in the docs on the use of selection, condition, and params with the `@vlplot` could be enough to have new users up to date with VegaLite interactivity.

**Comment by Jollywatt** [comment]
> Also, a section in the docs on the use of selection, condition, and params

I agree. I am a new user, and the lack of documentation around this is a problem — especially because there doesn’t seem to be any way to see errors caused by invalid Vega-Lite objects (instead, plots just appear blank) so trial and error is blind.
